nativelink-util/Cargo.toml

```diff
@@ -17,7 +17,9 @@ base64 = { version = "0.22.1", default-features = false, features = ["std"] }
 bitflags = { version = "2.9.0", default-features = false }
 blake3 = { version = "1.8.0", features = ["mmap"], default-features = false }
 bytes = { version = "1.10.1", default-features = false }
-futures = { version = "0.3.31", default-features = false }
+futures = { version = "0.3.31", features = [
+  "async-await",
+], default-features = false }
```

Comment:
I'm guessing fred pulled this in before, but need to spell it out explicitly here now